app/assets/stylesheets/admin.scss

```diff
@@ -188,3 +188,16 @@ label {
 .data_update_script__status {
   min-width: 120px;
 }
+
+// Navbar links have text-decorations that are being
```

Comment:
(request / blocking) - Not related to this particular CSS but I notice we don't have any focus styles for these links - making it inaccessible for keyboard users - can we add some here?

Reply:
@ludwiczakpawel I'm trying to get the focus styles for admin working. After some investigation I noticed [this css](https://github.com/forem/forem/blob/master/app/assets/stylesheets/scaffolds.scss#L16-L18). I know that you probably didnt add this, but do you by any chance  know why it could have been added ? It seems like if i comment this out, the focus works as expected, but i dont want to break anything unexpected. This gave me some more clues https://css-tricks.com/the-focus-visible-trick/

Reply:
> Not related to this particular CSS but I notice we don't have any focus styles for these links - making it inaccessible for keyboard users - can we add some here?
✅
Thanks to Pawel for helping me on this. I've added focus styles on the sidebar specifically - The focus elements on the admin interface as a whole needs to be fixed but I kept this change constrained to the feature I'm working on at the moment to avoid introducing bugs.